basic_bioinf_tools.py

```diff
@@ -0,0 +1,113 @@
+from bioinf_modules import dna_rna_tools
+from bioinf_modules import fastq_filter
+from bioinf_modules import protein_tools
+
+
+def run_dna_rna_tools(*args):
+    procedure, sequences = args[-1], args[:-1]
+    if procedure not in dna_rna_tools.PROCEDURES_DICT.keys():
+        raise ValueError("Wrong procedure")
+    rna_dna_tools_result_list = []
```

Comment:
Да просто было бы достаточно
```suggestion
    results = []
```